### PR TITLE
readme edits

### PR DESCRIPTION
**Pull request by elia314**
I updated/edited sections below. I didn't quite know what the badges/contributors/say thanks that were previously listed at top, so I created new section of Updates/credits to address those. 

I also didn't know what "Tools I used" section was for, was that helpful for this project or tools that users should look at if they want to use other tools?

@alejandronanez

### CODE REVIEW

**Comment by mubaris** [comment]
This is not a chrome extension. Put the badges on the top. I'll close this, and I'll explain what you need to do in #37

**Comment by alejandronanez** [comment]
Any chance we can reopen this? Per my comment in https://github.com/curiositylab/curiosity/issues/37#issuecomment-307568235

@elia314 we set the reviewers, don't worry about that 👍

**Comment by alejandronanez** [comment]
@elia314 let us know when you're done with the editing! Thanks for your contribution!

File: README.md [hunk 1]
```diff
@@ -29,7 +33,12 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
 
 ## What you need
 
-You need to submit Github Token to access Github API.
+You need to submit Github Token to access Github API. Do this by generating a personal token listed in your own settings at: https://github.com/settings/tokens
+
+## How to use
+1) Add Chrome Extension
```
Comment (maiquynhtruong) [inline README.md]: You don't need to add Chrome extension because this is not a Chrome extension

File: README.md [hunk 2]
```diff
@@ -29,7 +33,12 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
...
+2) Granted Curiosity your own github token
```
Comment (maiquynhtruong) [inline README.md]: I would reword it as: Submit your Github Token (generate it [here](https://github.com/settings/tokens)

File: README.md [hunk 3]
```diff
@@ -29,7 +33,12 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
...
+3) Open up a new Chrome browser tab, you'll see a feed of Github projects which are being used heavily (starred).
```
Comment (maiquynhtruong) [inline README.md]: I would reword it as: Open a new browser tab and run the `index.html` file in the root folder

File: README.md [hunk 4]
```diff
@@ -29,7 +33,12 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
...
 
-You need to submit Github Token to access Github API.
+You need to submit Github Token to access Github API. Do this by generating a personal token listed in your own settings at: https://github.com/settings/tokens
+
+## How to use
+1) Add Chrome Extension
+2) Granted Curiosity your own github token
+3) Open up a new Chrome browser tab, you'll see a feed of Github projects which are being used heavily (starred). 
 
 ## Tools I used
```
Comment (maiquynhtruong) [inline README.md]: You could change it to something like `Dependencies`

**Comment by elia314** [comment]
Closing this PR, follow PR #55 instead.
